angular-cli/base/Dockerfile

```diff
@@ -9,7 +9,7 @@ ARG CI_BUILD_ID
 ARG CI_BUILD_REF
 ARG CI_REGISTRY_IMAGE
 ARG CI_BUILD_TIME
-ARG NG_CLI_VERSION=latest
```

Comment:
please don't change this, use it as default

Reply:
ok fixed